models/User.php

```diff
@@ -2,103 +2,248 @@
 
 namespace app\models;
 
-class User extends \yii\base\BaseObject implements \yii\web\IdentityInterface
+use Yii;
+use yii\db\Expression;
+use yii\db\ActiveRecord;
+use yii\web\IdentityInterface;
+use app\models\Task;
+
+/**
+ * This is the model class for table "user".
+ *
+ * @property int $user_id
+ * @property string $name
+ * @property string $email
+ * @property string $password
+ * @property string|null $avatar
+ * @property string $date_add
+ * @property int|null $city_id
+ * @property string|null $phone
+ * @property string|null $telegram
+ * @property string|null $personal_information
+ * @property int|null $rating
+ * @property string|null $status
+ * @property string|null $birthday
+ * @property string $role
+ *
+ * @property Category[] $categories
+ * @property City $city
+ * @property Respond[] $responds
+ * @property Task[] $tasks
+ * @property Task[] $tasks0
+ * @property Task[] $tasks1
```

Comment:
Переменные типа `tasks0`, `tasks1` в коде лучше не держать. По имени переменной не ясно, что это за списки заданий

```suggestion
 * @property Task[] $customerTasks
 * @property Task[] $executorTasks
 * @property Task[] $respondedTasks
```